app/javascript/packs/hello_react.jsx

```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
```

Comment:
Do we need this file?

Reply:
This is example file generated when you create new app or initialize webpacker. When I made Rails upgrade I was trying to leave things as they generated by default to make minimum of changes.

Sure, we can remove this or even make some other 'demo' component, but I think in "base" project it's better to leave it untouched.